draft-birkholz-cose-cometre-ccf-profile.md

```diff
@@ -38,26 +38,125 @@ author:
   country: UK
 
 normative:
+  RFC9162: certificate-transparency-v2
 
 informative:
+  I-D.draft-steele-cose-merkle-tree-proofs: COMTRE
 
 --- abstract
 
-Abstract Text
+This document describes a tree algorithm for COSE Signed Merkle Tree Proofs specifically designed for implementations that rely on Trusted Execution Environments (TEEs) to make the
+ledger tamper-proof.
```

Comment:
```suggestion
log tamper-proof.
```

Reply:
Addressed both in 8a40a20. I also changed to tamper-resistant, I think I agree with your point